### PR TITLE
add the process to use the jurinet connector in the readme

### PR DESCRIPTION
**Pull request by fondation451**
## Changes

Explanation to run Jurinet locally

### CODE REVIEW

File: packages/courDeCassation/jurinetConnector/README.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+
+To import all the documents from Jurinet, run the following cron: `./src/crons/importAllDocumentsFromJurinet.ts`
+
+## Local version
+
+In order to be able to run LABEL locally, it is possible to use a test dataset stored locally.
+To do so, you need to put all your Jurinet court decisions in a folder `./storage/` at the root of the
+package.
+
+To run the local version of the connector, you to use node with the proper environment variable set `RUN_MODE=LOCAL`.
```
Comment (BenoitSerrano) [inline packages/courDeCassation/jurinetConnector/README.md:15]: word missing

File: packages/courDeCassation/jurinetConnector/README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+The Jurinet connector is the interface we have with the Jurinet Database.
+
+## Importation
+
+To import all the documents from Jurinet, run the following cron: `./src/crons/importAllDocumentsFromJurinet.ts`
+
+## Local version
+
+In order to be able to run LABEL locally, it is possible to use a test dataset stored locally.
+To do so, you need to put all your Jurinet court decisions in a folder `./storage/` at the root of the
```
Comment (BenoitSerrano) [inline packages/courDeCassation/jurinetConnector/README.md:12]: any extension of the files will do, or does it have to be a .xml ? Any name will do?

File: packages/courDeCassation/jurinetConnector/README.md (hunk 1 above)
Comment (fondation451) [reply packages/courDeCassation/jurinetConnector/README.md]: @BenoitSerrano  done a1d540825514b05e29c19a4a487c35a9f8f91fd2

File: packages/courDeCassation/jurinetConnector/README.md (hunk 2 above)
Comment (fondation451) [reply packages/courDeCassation/jurinetConnector/README.md:12]: @BenoitSerrano  done 83251377866b541774d7fd2985d05e87cff749c1

**Comment by fondation451** [comment]
@BenoitSerrano I have also updated the gitignore 99249789ad8ef9195c34db6f395f629d30a4e533

**Review by BenoitSerrano — APPROVED** [review]

**Pull request merged**